apps/desktop/src/styles.css

```
@@ -0,0 +1,1718 @@
+  place-items: center;
+  border: 1px solid var(--border);
+  border-radius: 8px;
+  color: var(--accent-text);
+  background: var(--accent-soft);
+}
+.setting-icon svg {
+  width: 15px;
+  height: 15px;
+}
+.setting-card p {
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)